src/samples/EventArray.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef __BASE_SAMPLES_EVENT_ARRAY_HH
+#define __BASE_SAMPLES_EVENT_ARRAY_HH
+
+#include <base/samples/Event.hpp>
+
+namespace base {
+namespace samples {
+
+    struct EventArray
+    {
+        ::base::Time time;
+        uint16_t height;
+        uint16_t width;
+
+        std::vector<Event> events;
```

Comment:
Missing `#include <vector>`

Reply:
Updated. Sorry for missing it!

Reply:
No worries at all ... This is why we have reviews ;-)